apps/system/js/nfc_handover_manager.js

```diff
@@ -161,6 +161,11 @@
       this.bluetoothStatusSaved = false;
       this.bluetoothAutoEnabled = false;
 
+      if (typeof(window.navigator.mozBluetooth.onattributechanged) !==
+        'undefined') {
+        this._debug('Bluetooth APIv2 does not support NFC yet');
+        return;
```

Comment:
Basically the patch will not break BTv1 and make master works with BTv2.
The patch is the 2nd step to introduce BTv2 into system. The followup should (3rd)add BT transfer and (4th)nfc_handover functionality for BTv2.